README.md

```diff
@@ -20,7 +20,7 @@ Supported versions:
 *  Kubernetes 1.12+ or OpenShift 3.11+
 *  Elastic Stack: 6.8+, 7.1+
 
-Check the [Quickstart](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html) if you want to deploy you first cluster with ECK.
+Check the [Quickstart](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html) if you want to deploy your first cluster with ECK.
```

Comment:
```suggestion
Check the [Quickstart](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html) to deploy your first cluster with ECK.
```

Good catch, though now that I'm looking at this sentence I think we can make it more active